lib/cisco_node_utils/cmd_ref/acl.yaml

```diff
@@ -0,0 +1,33 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
+  config_get_token: '/^<afi> access-list <acl_name>$/'
+  config_set: '<afi> access-list <acl_name>'
+
+ace:
+  config_get_token_append: '/^<seqno> .+$/'
+  config_set_append:
+      '<state> <seqno> <action> <proto> <src_addr> <src_port> <dst_addr> <dst_port> <option_format>'
```

Comment:
I'm surprised but okay. Please move it up to the config_set_append line anyway so that I can find it when I grep through yaml files.